csrc/scheduler/normalization_utils.h

```diff
@@ -191,5 +191,16 @@ int64_t getInnerOuterPersistentBufferBatches(
     const int64_t outer_dim_numel,
     const int64_t warpSize);
 
+//! Each thread can use a maximum of 255 registers, and assume 40 of them are
+//! reserved for indexing and other purposes. So, each thread can use up to 215
+//! registers for persistent buffer. Calculate number of buffer batches using
```

Comment:
Doesn't the function just return the register count? Is it equivalent to the number of buffer batches?

Reply:
Oops, sorry, my misunderstanding.